src/sass/App.scss

```diff
@@ -0,0 +1,13 @@
+$fa-font-path: "~font-awesome/fonts";
+
+@import "../../node_modules/@edx/edx-bootstrap/sass/edx/theme";
```

Comment:
I'm not 100% sure of the syntax of this, but I think `"~@edx/edx-bootstrap..."` should do all the pathing for you for this? I guess this is what you have to do if that doesn't work.

Reply:
Not sure, i actually didn't write this but just moved the file.